Formula/sing-box.rb

```diff
@@ -19,11 +19,18 @@ class SingBox < Formula
   depends_on "go" => :build
 
   def install
+    (etc/"sing-box").mkpath
+
     ldflags = "-s -w -X github.com/sagernet/sing-box/constant.Version=#{version} -buildid="
     tags = "with_gvisor,with_quic,with_wireguard,with_utls,with_reality_server,with_clash_api"
     system "go", "build", "-tags", tags, *std_go_args(ldflags: ldflags), "./cmd/sing-box"
   end
 
+  service do
+    run [opt_bin/"sing-box", "run", "-D", etc/"sing-box", "-c", "config.json"]
```

Comment:
Does a user need to provide their own `config.json` for `brew services start sing-box` to work?

Reply:
Where does it read the config from?

Reply:
According to the description of the sing-box project, what I want is to create a sing-box directory under etc, and then require users to manually create a config.json configuration file in this directory, and then read this config.json.

But I realize two other problems:
1. -D should not point to `etc/"sing-box"`, because -D refers to the working directory, and sing-box will download some files after startup, such as geosite.db, geoip.db and so on. If you do not specify the working directory, the default is the path of the current command execution. For example, under `HOME`, the downloaded file will appear in `HOME`. I don’t know where it will be downloaded after passing `service run or start`. So where should the working directory be set to? Any suggestions or specifications?
2. In the previous 1.2-beta10 version, multiple configuration files were supported. For details, please refer to:
https://sing-box.sagernet.org/changelog/#12-beta10
    > Now you can pass the parameter --config or -c multiple times, or use the new parameter --config-directory or -C to load all configuration files in a directory.

    > Loaded configuration files are sorted by name. If you want to control the merge order, add a numeric prefix to the file name.

    So my idea is to create etc/sing-box configuration directory. Then use the -C parameter to specify it.used to support this feature.

Hope to get your help, thank you